modis_lai_fluxnet_sites/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+ClimaArtifactsHelper = "6ffa2572-8378-4377-82eb-ea11db28b255"
+ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
```

Comment:
We ended up not needing `ProgressMeter`, so you can remove it from the environment and re-resolve the Manifest